USE-CASES/hazard-annotations.md

```diff
@@ -0,0 +1,135 @@
+- A CO₂/VOC sensor may provide a default policy that modulate the smoke and fire hazard evaluation based on its own information, but in itself may not detect smoke or temperature.
+- A connected Oven may not have a smart meter embedded but may provide enough information to desume the energy consumption by the temperature of its heating elements.
+
+### Dependencies - Affected WoT deliverables and/or work items:
+
+The current Thing Description is adequate to support the model, the hazards annotations can be bound to the specific Form Operation for each Interaction Affordance.
+Additional vocabulary terms may be needed to support a finer level of granularity or declare default policies or formulas to derive additional hazard based on the aggregation of information provided by different devices..
+
+### Description:
+
+The user may have a mean of setting policies to restrict interactions with connected devices, e.g.:
+- Stop any interaction with the TV between 3:00PM and 5:00PM if an unattended children is detected.
+- Require stronger authentication to open a door if the user is considered already inside the house.
+
+The home system may automously change the state of a device to reduce the risk level, e.g.:
+- Reduce the brightness of an Halogen Lamp
+- Start the A/C system if a room temperature is outside the bearable range and a person is detected inside.
+
+The same considerations apply to larger scale (e.g. apartment buildings, neighbourhoods, cities) or slightly different contexts such as industry or hospitals.
+
+The risk categories may be diversified to cover field-specific risks, but the structure remains the same:
+- The Thing Description would provide the hazard information
+- Consumer may act on it or enrich/aggregate it and pass it forward for further processing
+
+### Security Considerations:
+
+The system relies on being able to trust the information exchanged and has to detect bad actors.
+
+### Privacy Considerations:
+
+Privacy is one of the possible category of hazards that may be accounted for, it is no different from a physical hazard in this use-case.
+
+### Accessibility Considerations:
+
+The system may be extended to cover accessibility concerns and have specific policies.
+
+### Internationalisation (i18n) Considerations:
+
+Being also user-facing, the hazard information should be translated.
+
+### Requirements:
+
+The Thing Description does not have specific fields to deliver all the information described, but the existing structure can be augmented through additional vocabulary terms.
+Any interaction among Things and Consumers is within the normal operation and should not require changes.
+
+#### Protocol Requirements
+
+"flexible"
+
+#### Content Type Requirements
+
+"flexible"
+
+#### Platform or Standard Requirements
+
+"none"
+
+#### Authentication and Authorization Mechanisms Requirements
+
+"flexible"
+
+### Gaps:
+
+### Existing standards:
```

Comment:
```suggestion
### Existing standards:

There are many existing standards for various types of hazards in specific industries and
locals, for example for electrical or chemical hazards.  There are also in many locales legal
requirements for package labelling and in-package notifications, e.g. for chemical, electrical,
or ergonomic hazards.  However, this information needs to be presented in a common
electronic format that is accessible to e.g. home automation systems.  It may be necessary
to develop a new ontology for general categories of hazards that are location-idenpendent,
but this should be based on existing hazard labelling whenever possible.
```